lib/cocoapods-core/specification/dsl.rb

```diff
@@ -867,12 +867,17 @@ def dependency(*args)
       #   @example
       #
       #     spec.prefix_header_file = 'iphone/include/prefix.pch'
-      #     spec.prefix_header_file = ':skip'
       #
-      #   @param  [String] path
-      #           The path to the prefix header file.
+      #   @example
+      #
+      #     spec.prefix_header_file = false
+      #
+      #   @param  [Bool, String] path
+      #           The path to the prefix header file or whether to disable
+      #           prefix_header generation.
       #
       attribute :prefix_header_file,
+                :types => [TrueClass, FalseClass, String],
```

Comment:
What will setting it to true do?

Reply:
I added true for symmetry with false. If it is set, it will behave the same as if the option is not set at all and the default CocoaPods pch file will be generated. 

See this test: https://github.com/paulb777/CocoaPods/blob/68abbb06a4a2d5adf9c5ce6361fab78af60be8f5/spec/unit/installer/xcode/pods_project_generator/pod_target_installer_spec.rb#L412

Reply:
👍 but we should doc it